src/Commands/BaseCommand.php

```diff
@@ -92,7 +92,7 @@ protected function getFileContent(InputInterface $input, $option)
         $filename = $input->hasOption($option) ? $input->getOption($option) : 'php://stdin';
 
         if (!file_exists($filename)) {
-            return $filename;
+            throw new \InvalidArgumentException('The file provided "'.$filename.'" was not found.');
```

Comment:
This actually needs to stay as it was previously; I'm using this in [`src/Commands/Recipes/Make.php`]() to allow the user to supply a filename (`recipe:make --script=./bin/myscript.sh`), the script as a string (`--script="echo 'hi'"`), or directly from `STDIN` (`recipe:make < ./bin/myscript.sh`). 

Can I ask what prompted this change? What did you want to do, and what did you expect to happen?

Reply:
Well, the code seems like it wants a filename. If it's not valid, then it's not a valid filename. In other projects I've worked with, if you want a filename, you give it a filename... otherwise you feed the data in through another method. This just seems cleaner.

But if you give it an invalid filename and it doesn't fail, then your recipe will just contain a string to an invalid file.